src/main/java/com/ecore/roles/client/model/User.java

```diff
@@ -1,38 +1,27 @@
 package com.ecore.roles.client.model;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
+import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.Setter;
 
 import java.util.UUID;
 
 @NoArgsConstructor
 @AllArgsConstructor
-@Getter
-@Setter
 @Builder
-@EqualsAndHashCode
+@Data
 public class User {
 
-    @JsonProperty
```

Comment:
If we aren’t customizing the json serialization, then we don’t need to add the @JsonProperty annotation